kratos/python_scripts/modelers/embedded_geometry_modeler.py

```diff
@@ -0,0 +1,56 @@
+import KratosMultiphysics as KM
+
+class EmbeddedGeometryModeler(KM.Modeler):
+
+    def __init__(self, model, settings):
+        super().__init__(model, settings)
+
+        # Cannot validate as settings may differ ammong input types
+        settings.AddMissingParameters(self.__GetDefaultSettings())
+
+        # Declare required member variables
+        self.model_part = None
+        self.settings = settings
+
+        # Get the model parts
+        self.volume_model_part = model.CreateModelPart(settings["volume_model_part_name"].GetString())
+        self.embedded_model_part = model.CreateModelPart(settings["embedded_model_part_name"].GetString())

```

Comment:
```suggestion
        self.model_part = model.CreateModelPart(settings["model_part_name"].GetString())
        self.skin_model_part = model.CreateModelPart(settings["skin_model_part_name"].GetString())

```
To be consistent to the usual notation in Kratos.